definitions/infra-harbor/summary_metrics.yml

```diff
@@ -0,0 +1,27 @@
+httpLiveTraffic:
+  goldenMetric: httpLiveTraffic
+  title: "Current (in-flight) HTTP requests"
+  unit: COUNT
+  queries:
+    prometheus:
+      select: latest(harbor_core_http_inflight_requests)
+      from: Metric
+
+httpTraffic:
+  goldenMetric: httpTraffic
+  title: "Sum of HTTP requests over a time period"
+  unit: COUNT
+  queries:
+    prometheus:
+      select: sum(harbor_core_http_request_total)
+      from: Metric
+      facet: operation
```

Comment:
One last thing, the `facet` is not allowed in the summary metric definition.

Reply:
done